app/controllers/conversion/projects_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Conversion::ProjectsController < ProjectsController
+  def new
+    authorize Conversion::Project, policy_class: ProjectPolicy
```

Comment:
Do we need to cover this in the request specs, like this https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/5d1f9c4552a94a8d8341b8eaa8d63dbd301c7fc4/spec/requests/notes_controller_spec.rb#L277

It makes sense to me that the unauthorised behaviour would be tested there 🤔, but maybe somewhere else?

Reply:
Added!